src/collections/_documentation/workflow/alerts-notifications/alerts.md

```diff
@@ -0,0 +1,187 @@
+    - "service" is the same as legacy integration
+- Send a notification to a global integration, which includes
+    - [PagerDuty](https://docs.sentry.io/workflow/integrations/global-integrations/#pagerduty)
+    - [Slack](https://docs.sentry.io/workflow/integrations/global-integrations/#slack)
+    - Custom integrations built using the [Integration Platform](https://docs.sentry.io/workflow/integrations/integration-platform/)
+
+![DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-25_15.28.01.png](DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-25_15.28.01.png)
+
+- Legacy Integrations (via “Service” as any individual integration) that can send Notifications:
+
+    [ ALERT BOX: **Community Integrations** 
+
+    These integrations are [maintained and supported](https://forum.sentry.io/) by the Sentry community. ]
+
+    - Campfire*
+    - IRC
+    - OpsGenie
+    - Pushover
+    - Twilio
+    - VictorOps
+    - [Webhooks](https://docs.sentry.io/webhook-plugin/)
+
+By default, [LINK: issue owners] receive alert emails. If an issue owner is not configured or found, the email will either not send or send to all project members as defined in **Project Settings > Issue Owners**.
+
+![DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-26_22.03.07.png](DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-26_22.03.07.png)
+
+**Rate Limit**
+
+Rate Limit control allows you to limit the number of times actions execute for each issue. The limit is set to perform the action according to one of these intervals:
+
+- minutes: 5, 10, 30, 60
+- hours: 3, 12, 24
+- one week or 30 days
+
+For example, if an issue violates alert conditions multiple times in a 30-min period, but your frequency threshold is 30 minutes, you’ll only get one alert.
+
+![https://docs.sentry.io/assets/notifications/alert_frequency-40d4d353342465f52d23147ed2fb32986e1a502c889cfb5579723cd161b53a62.png](https://docs.sentry.io/assets/notifications/alert_frequency-40d4d353342465f52d23147ed2fb32986e1a502c889cfb5579723cd161b53a62.png)
+
+### Environment
+Environment control allows you to specify which environment qualifies for your issue alert.
+
+- All Environments
+- Any one of your defined (and not hidden) environments
+
+![https://docs.sentry.io/assets/notifications/alert_environment-e6e58d46f0a5773deb78f740f1f14ab37f236699161492dc20023d6586504529.png](https://docs.sentry.io/assets/notifications/alert_environment-e6e58d46f0a5773deb78f740f1f14ab37f236699161492dc20023d6586504529.png)
+
+If you select "All Environments", Sentry checks individually for each environment, rather than combined events across environments.
+
+### Digests
+
+The digests feature works only for **issue alert emails**. This project-level setting allows you to batch and deliver issue alert emails for a project. Use the sliders to control the frequency.
```

Comment:
"This project-level setting allows you to batch issue alerts to limit the total number of emails you receive for that project."